avocado/plugins/spawners/podman.py

```diff
@@ -273,6 +273,8 @@ async def spawn_task(self, runtime_task):
         container_id = await self._create_container_for_task(
             runtime_task, env_args, output_dir_path
         )
+        if container_id is False:
```

Comment:
What about to use exceptions here. Somethink like:
```
--- a/avocado/plugins/spawners/podman.py
+++ b/avocado/plugins/spawners/podman.py
@@ -236,21 +236,16 @@ class PodmanSpawner(DeploymentSpawner, SpawnerMixin):
             image = self.config.get("spawner.podman.image")

         envs = [f"-e={k}={v}" for k, v in env_args.items()]
-        try:
-            # pylint: disable=W0201
-            _, stdout, _ = await self.podman.execute(
-                "create",
-                *status_server_opts,
-                *output_opts,
-                *test_opts,
-                entry_point_arg,
-                *envs,
-                image,
-            )
-        except PodmanException as ex:
-            msg = f"Could not create podman container: {ex}"
-            runtime_task.status = msg
-            return False
+        # pylint: disable=W0201
+        _, stdout, _ = await self.podman.execute(
+            "create",
+            *status_server_opts,
+            *output_opts,
+            *test_opts,
+            entry_point_arg,
+            *envs,
+            image,
+        )

         return stdout.decode().strip()

@@ -270,9 +265,15 @@ class PodmanSpawner(DeploymentSpawner, SpawnerMixin):
         destination_eggs = ":".join(map(lambda egg: str(egg[1]), eggs))
         env_args = {"PYTHONPATH": destination_eggs}
         output_dir_path = self.task_output_dir(runtime_task)
-        container_id = await self._create_container_for_task(
-            runtime_task, env_args, output_dir_path
-        )
+        try:
+            container_id = await self._create_container_for_task(
+                runtime_task, env_args, output_dir_path
+            )
+
+        except PodmanException as ex:
+            msg = f"Could not create podman container: {ex}"
+            runtime_task.status = msg
+            return False

```

Reply:
Sure, no objections to your proposal.